client/state/sites/selectors.js

```diff
@@ -145,7 +170,7 @@ export function isJetpackMinimumVersion( state, siteId, version ) {
  * @return {?String}        Site slug
  */
 export function getSiteSlug( state, siteId ) {
-	const site = getSite( state, siteId );
+	const site = getRawSite( state, siteId );
```

Comment:
So from memory, I envisaged composing the selectors as the best solution, eventually getting rid of legacy computed attributes when we nuke `sites-list` — hence the redundancy for now.